x-pack/test/cases_api_integration/security_and_spaces/tests/common/comments/delete_comment.ts

```diff
@@ -91,9 +111,299 @@ export default ({ getService }: FtrProviderContext): void => {
       });
     });
 
-    describe('rbac', () => {
-      const supertestWithoutAuth = getService('supertestWithoutAuth');
+    describe('alerts', () => {
+      describe('security_solution', () => {
+        const createCaseAttachAlertAndDeleteAlert = async ({
+          totalCases,
+          indexOfCaseToDelete,
+          expectedHttpCode = 204,
+          auth = { user: superUser, space: 'space1' },
+        }: {
+          totalCases: number;
+          indexOfCaseToDelete: number;
+          expectedHttpCode?: number;
+          auth?: { user: User; space: string | null };
```

Comment:
How about we rename this to `deleteCommentAuth` since I think that's the only time we're using it right?